src/Models/ACF/Page/Widgets/SeoText.php

```diff
@@ -99,4 +100,32 @@ private function getImageField()
             'mime_types' => '',
         ];
     }
+
+    public function getImagePosition()
+    {
+        return [
+            'key' => 'field_5be56c5ca42ef',
+            'label' => 'Image Position',
+            'name' => 'image_position',
+            'type' => 'radio',
+            'instructions' => '',
+            'required' => 0,
+            'conditional_logic' => 0,
+            'wrapper' => [
+                'width' => '',
+                'class' => '',
+                'id' => '',
+            ],
+            'choices' => [
+                'left' => 'Left',
+                'right' => 'Right',
```

Comment:
Maybe it should be "before / after text".
What if we implement a new design for another brand, where the image should be above or below?

Reply:
Maybe it's to being to pedantic, what is your oppinion, @alfhen ?